### PR TITLE
8269 dtrace stddev aggregation is normalized incorrectly

### PR DESCRIPTION
**Pull request by avg-I**
https://www.illumos.org/issues/8269

### CODE REVIEW

File: usr/src/cmd/dtrace/test/tst/common/aggs/tst.stddev.normalize.d [hunk 1]
```diff
@@ -0,0 +1,58 @@
+/*
```
Comment (yuripv) [inline usr/src/cmd/dtrace/test/tst/common/aggs/tst.stddev.normalize.d]: Please use license text from usr/src/prototypes/ for new files.

File: usr/src/cmd/dtrace/test/tst/common/aggs/tst.stddev.normalize.d [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+ * information: Portions Copyright [yyyy] [name of copyright owner]
+ *
+ * CDDL HEADER END
+ */
+
+/*
+ * Copyright 2017 Panzura.  All rights reserved.
+ */
+
+#pragma	ident	"%Z%%M%	%I%	%E% SMI"
```
Comment (yuripv) [inline usr/src/cmd/dtrace/test/tst/common/aggs/tst.stddev.normalize.d]: Drop this.

**Comment by avg-I** [comment]
@yuripv, done. Thank you.

**Comment by prakashsurya** [comment]
@avg-I can you rebase this on the latest master branch? I broke the build last night, but I just pushed a commit to fix the breakage; see https://github.com/openzfs/openzfs/commit/452fdc03c471f4831e5e509fcba941a46968f260

**Comment by prakashsurya** [comment]
this looks good with Bryan's review; the mail was just sent to the illumos developer list, so I'll wait a couple days to see if anybody else chimes in, if not I'll open the RTI.

**Comment by prakashsurya** [comment]
this was pushed to illumos, we'll pick it up in the next illumos sync. see here: https://github.com/illumos/illumos-gate/commit/79809f9cf402f130667349b2d4007ecd65d63c6f

**Pull request closed without merging**